lfucg-exactions/server/server/settings/base.py

```diff
@@ -310,4 +310,5 @@ def custom_show_toolbar(request):
 #CEF_VENDOR = 'Your Company'
 #CEF_VERSION = '0'
 #CEF_DEVICE_VERSION = '0'
-PAGINATION_SIZE = 10
\ No newline at end of file
+PAGINATION_SIZE = 10
+PASSWORD_RESET_TIMEOUT_DAYS = 1
```

Comment:
Check if they are okay with the reset links being active for a day and the token being deleted after a successful password change. If not, you can read this for ideas. http://pminkov.github.io/blog/how-to-generate-user-activation-links-in-django.html

Reply:
Ok thanks! We are checking with Craig and will see if that is good or not.